Godot.gitignore

```diff
@@ -0,0 +1,8 @@
+temp/
+.import/
+.mono/
+*.import
+*.DS_Store
+*.directory
```

Comment:
`.DS_Store` and `.directory` are files, not extensions, so no need to put a leading `*`.

They're also not Godot-related, they come respectively from macOS's Finder and KDE's Dolphin, so I don't know if they're meant to be included in the Godot template (haven't checked how this gitignore repo is used exactly).